packages/plugin-kafka/__tests__/kafka.spec.ts

```diff
@@ -0,0 +1,197 @@
+/**
+ * @jest-environment node
+ */
+
+// tests long running
+jest.setTimeout(10000)
```

Comment:
its best to move it over to jest.config.js

Reply:
jest.config does not support this option. created plugins own jest config that inherits base config. And added env setup script. This option is almost useless on powerful workstations, but for CI or a bit slow pcs, solves probem when kafka rebalances clients.